examples/example.py

```diff
@@ -46,14 +54,30 @@ def do_speak(self, arg, opts=None):
             arg = arg.upper()
         repetitions = opts.repeat or 1
         for i in range(min(repetitions, self.maxrepeats)):
-            self.stdout.write(arg)
-            self.stdout.write('\n')
-            # self.stdout.write is better than "print", because Cmd can be
-            # initialized with a non-standard output destination
+            self.poutput(arg)
+            # recommend using the poutput function instead of
+            # self.stdout.write or "print", because Cmd allows the user
+            # to redirect output
 
     do_say = do_speak  # now "say" is a synonym for "speak"
     do_orate = do_speak  # another synonym, but this one takes multi-line input
 
+    @options([ make_option('-r', '--repeat', type="int", help="output [n] times") ])
+    def do_mumble(self, arg, opts=None):
```

Comment:
I like this addition.  It is a convenient way to introduce motivation for using regexes within transcripts due to unpredictable randomness.